GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -911,9 +911,9 @@ private static String dateCellValue(String value) {
         Media media = MediaResponse.parse(value);
         String filename = media.getFilename();
         if (filename != null) {
-            final int filenameIndex = filename.lastIndexOf("/") + 1;
-            if (filenameIndex > 0 && filenameIndex < filename.length()) {
-                cells.add(imagePrefix + filename.substring(filenameIndex));
+            filename = filename.substring(filename.lastIndexOf("/") + 1); //strip path, if any
```

Comment:
I would suggest to use `Paths.get(filename).getFileName()` instead of doing _substring_ manipulation.
`java.nio.file.Paths` was introduced in Java 7, so is safe to use.
```java
Paths.get("/this/is/a/long/path.txt").getFileName().equals(Paths.get("path.txt").getFileName());
// true
Paths.get("/this/is/a////long/path.txt").getFileName().equals(Paths.get("path.txt").getFileName())
// true
```

Reply:
Done